core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -749,6 +749,13 @@ class EMConfig {
     var extraHeuristicsFile = "extra_heuristics.csv"
 
 
+    @Cfg("Enable to print snapshots of the generated tests during the search in an interval defined in snapshotsInterval.")
```

Comment:
add `@Experimental`

Reply:
once we have used/tests this feature for a bit, then we can remove the `@Experimental` and put it `true` by default